src/slider.scss

```diff
@@ -16,11 +20,12 @@ $handle-hit-area-mobile-dimensions: 2.75rem;
 
   min-width: 4rem;
   max-width: 100%;
-  padding: 1rem 0.625rem;
+  padding: 1rem calc(#{$slider-x-paddings} + #{$handle-outline-offset} + #{$handle-outline-width});
   position: relative;
 
   &--lg {
-    @include fd-set-paddings-x-equal(0.8125rem);
+    padding-left: calc(#{$slider-x-paddings--lg} + #{$handle-outline-offset} + #{$handle-outline-width});
+    padding-right: calc(#{$slider-x-paddings--lg} + #{$handle-outline-offset} + #{$handle-outline-width});
```

Comment:
Could we use `padding: 1rem calc(#{$slider-x-paddings--lg} + #{$handle-outline-offset} + #{$handle-outline-width});`?

Reply:
Why? 1 rem for Y pending is already installed. We need to change just X pending.
But I have better idea to use fd-set-paddings-x-equal mixin for it.